src/datasource.ts

```diff
@@ -0,0 +1,300 @@
+import {
```

Comment:
Doesn't Apache expect you to put a license reference in the headers of sources?
https://www.apache.org/legal/src-headers.html

Reply:
Nah, that only applies to Apache projects, stuff that's published by ASF.